frontend/packages/metal3-plugin/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@console/metal3-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Metal Kubed - Bare Metal Host Provisioning for Kubernetes",
+  "private": true,
+  "dependencies": {
+    "@console/plugin-sdk": "0.0.0-fixed",
+    "@console/shared": "0.0.0-fixed",
+    "kubevirt-web-ui-components": "^0.1.34-2"
```

Comment:
Note: you can evaluate the impact of adding `kubevirt-web-ui-components` on webpack-generated JS dist code by running `yarn analyze` which runs the [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer).